content/en/cd-as-a-service/concepts/iam/rbac.md

```diff
@@ -0,0 +1,323 @@
+
+You define your roles in a YAML file that has this structure:
+
+{{< prism lang="yaml" line-numbers="true" >}}
+roles:
+  - name: <role-name>
+    tenant: <tenant-name>
+    grants:
+      - type: <type>
+        resource: <resource>
+        permission: <permission>
+{{< /prism >}}
+
+You can create an organization-wide role by omitting the `tenant` definition.
+
+### Grants
+
+A _Grant_ has type, resource, and permission attributes.
+
+`type` has a single choice: `api`. 
+
+`resource` has the following values:
+
+* `tenant`: When you use `tenant` as the `resource`, the Grant allows access to the tenant that you specify in the `roles.tenant` field. You use `tenant` when you define a [Tenant Admin role](#tenant-admin-role).
+* `deployment`: This resource allows the role to deploy using the CLI and manage deployments in the **Deployments** UI. If you omit `roles.tenant`, the role has this Grant across your organization.
+* `organization`: You use this resource when you need to create an Organization Admin role that maps to an SSO group. See [SSO support](#sso-support) for more on mapping SSO groups to RBAC roles.
+
+`permission` has one option: `full`.
+
+## Examples
+
+### Tenant Admin role
+
+This example defines three Tenant Admin roles, one for each tenant. Each role has full authority within the specified tenant.
+
+{{< prism lang="yaml" line-numbers="true" >}}
+roles:
+  - name: Tenant Admin Main
+    tenant: main
+    grants:
+      - type: api
+        resource: tenant
+        permission: full
+  - name: Tenant Admin Finance
+    tenant: finance
+    grants:
+      - type: api
+        resource: tenant
+        permission: full
+  - name: Tenant Admin Commerce
+    tenant: commerce
+    grants:
+      - type: api
+        resource: tenant
+        permission: full
+{{< /prism >}}
+
+If you want to grant a user permission to manage all of your tenants, assign that user the **Organization Admin** role using the UI.
+
+### Deployment roles
+
+This example defines a role that grants permission to use the **Deployments** UI and start deployments using the CLI. The role is bound to the `finance` tenant.
+
+{{< prism lang="yaml" line-numbers="true" >}}
+roles:
+  - name: Deployer
+    tenant: finance
+    grants:
+      - type: api
+        resource: deployment
+        permission: full
+{{< /prism >}}
+
+This example defines a role that grants permission to use the **Deployments** UI and start deployments using the CLI across your entire organization. Note that `tenant` is not defined, which makes this an organization-wide role.
+
+{{< prism lang="yaml" line-numbers="true" >}}
+roles:
+  - name: Deployer
+    grants:
+      - type: api
+        resource: deployment
+        permission: full
+{{< /prism >}}
+
+## SSO support
```

Comment:
Maybe this block should go below the "Assign roles to users"? If I'm reading this, I might think that you can only use RBAC if you have SSO enabled.